lib/tasks/link_checker.rake

```diff
@@ -1,28 +1,29 @@
 require "uri"
 
 def gather_urls object
-  urls = []
+  urls = Set[]
```

Comment:
Can you add note about why you're using `Set` to the commit message? What's the benefit?